faiss/impl/ScalarQuantizer.cpp

```diff
@@ -601,7 +601,7 @@ void train_NonUniform(
             }
         }
         std::vector<float> trained_d(2);
-#pragma omp parallel for
+#pragma omp parallel for if (d > 1)
```

Comment:
isn't this path hit during indexing?
I thought the focus was query path.

Reply:
yeah i observed that this file's line was hit around 30k times with n or d always being 1, during indexing.

so i just added the guard here as well

i added debug statements for every pragma statement in faiss in that separate branch, and every other pragma stmt that was hit was already guarded except the ones in this file

Reply:
cool, just wanted to raise this so we're sure which paths are being targeted. 

anytime da :)